React/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,35 @@
+import { isArray } from './utils';
+
+export function min(...args) {
+  if (args.length === 0) {
+    return undefined;
+  }
+  let spread = [];
+  args.forEach(pos => {
+    if (isArray(pos)) {
+      spread = [...spread, ...pos];
+    } else {
+      spread.push(pos);
+    }
+  });
+  return Math.min(...spread);
+}
+
+export function copy(element) {
+  if (isArray(element)) {
+    return [...element];
+  }
+  const { ...copied } = element;
+  return copied;
+}
+
+export function reverseMerge(array1, array2) {
+  return [...array2, ...array1];
+}
+
+export function filterAttribs(object) {
+  const { ...filtered } = object;
```

Comment:
You could do something better with a little thing that I told you yesterday,
If you do `{ a, b, ...filtered }`instead, it would leave the rest of the object properties on `filtered` and you wouldn't need to delete that props from the object you just copied!